embedded_gems/pathogen/app/components/pathogen/tabs_panel.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Pathogen
+  # This file defines the Pathogen::TabsPanel component, which is used to render sets of tabs
+  # as well as any inline 'right-side' components (buttons, filter, etc.)
+  class TabsPanel < Pathogen::Component
+    TAG_DEFAULT = :nav
+    TAG_OPTIONS = [TAG_DEFAULT, :div].freeze
+
+    TYPE_DEFAULT = 'default'
+    TYPE_OPTIONS = [TYPE_DEFAULT, 'underline'].freeze
+
+    BODY_TAG_DEFAULT = :ul
+    # rubocop:disable Layout/LineLength
+    BODY_DEFAULT_CLASSES = 'flex flex-wrap text-sm font-medium text-center text-slate-500 border-b border-slate-200 dark:border-slate-700 dark:text-slate-400'
```

Comment:
Instead of using rubocop disable here, just make a multi-line string like you do for the tests?
```suggestion
    BODY_DEFAULT_CLASSES = 'flex flex-wrap text-sm font-medium text-center ' \
                          'text-slate-500 border-b border-slate-200 ' \
                          'dark:border-slate-700 dark:text-slate-400'
```

Reply:
Agreed fixed in d5ab243eb9e156f1977ad5421babe58988681ff9